### DISCUSSION TITLE
Github Pro limitations

### DISCUSSION THREAD

**Discussion by Fuzzel11** [community/community#23297] [New to GitHub]
<p>I’m a student that got the student developer pack including Github Pro.</p>
<p>I was wondering what exactly the limitations of this plan are. There’s a lot of different information about what Github Pro is/was and I can’t find the Pro plan on the pricing charts anymore, so I’m not sure where to get up-to-date information on this plan.</p>
<p>I’m currently doing a master degree in game development and are soon going to start a bigger project with 6 people and we were looking around for a fitting VCS.</p>
<p>We found the “Github for Unity” integration which uses LFS, so I was wondering with my current Github Pro account, for a private repository, what exactly are the user limitations, repository size limitations and monthly bandwidth limitations (esp. in consideration that we will use LFS)?</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/linard11">@linard11</a>,</p>
<p>Thanks for checking. In addition to our pinned topic <a href="https://github.community/t/are-there-any-changes-to-github-pro/18546" title="Pinned Topic: Changes to GitHub Pro">Are there any changes to GitHub Pro?</a>, each GitHub account has a <a href="https://help.github.com/en/github/managing-large-files/about-storage-and-bandwidth-usage" title="GitHub Storage and Bandwidth Limits" rel="nofollow noopener">limit</a> of 1 GB of free storage and 1 GB of free bandwidth. If you believe you may need more than this, we’d recommend taking a look at our <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/upgrading-git-large-file-storage" title="Upgrading Storage and Bandwidth" rel="nofollow noopener">LFS Data Packs</a>, which provide 50 GB of storage and 50 GB of bandwidth per month for $5/each, and you can add as many data packs as you need.</p>
<p>Data packs are not covered by the Student Developer Pack, so do keep this in mind. If you have further questions about what your GitHub Student Developer Pack provides, also be sure to check on GitHub’s Education for Students <a href="https://education.github.com/pack#offers" title="GitHub Student Developer Pack Offers" rel="nofollow noopener">page</a>.</p>
